test/volume_test.go

```diff
@@ -22,38 +25,61 @@ type VolumeWorkflowSuite struct {
 	volcollService      *service.VolumeCollectionService
 	initiatorGrpService *service.InitiatorGroupService
 }
-
+func (suite *VolumeWorkflowSuite) SetupTest() {
+    fmt.Println("SetupTest")
+	if testResult == "PASS" && result == "PASS" {
+	testResult = "PASS"	
```

Comment:
This statement has no effect.  Same thing at line 58.

Reply:
refactored the code